tools/build/src/common/container-helper.ts

```diff
@@ -0,0 +1,73 @@
+/**********************************************************************
+ * Copyright (c) 2020-2021 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (c) 2021 Red Hat, Inc.
```

it was not there in 2020

Reply:
fixed